src/os/TempFile.hxx

```diff
@@ -129,7 +129,9 @@ public:
       HASSERT(ret >= 0);
       ofs += ret;
     }
+#ifndef __WINNT__    
```

Comment:
same.

Reply:
i moved this change out of this PR because it is unrelated.